examples/gallery/magnetogram_active_regions.py

```diff
@@ -0,0 +1,123 @@
+"""
+=========================================================
+Overplotting Active Region locations on magnetogram Plots
+=========================================================
+
+This example shows how to overplot Active Region location on magnetogram plots.
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+
+import datetime
+
+import matplotlib.pyplot as plt
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+
+from sunpy.io.special import srs
+import sunpy.coordinates
+from sunpy.net import Fido, attrs as a
+import sunpy.map
+from sunpy.time import parse_time
+
+##############################################################################
+# Let's select a date (yyyy-mm-dd) for which we will be downloading files.
+
+day = parse_time("2017-01-25")
+
+##############################################################################
+# We will select the entire day as our timerange.
+
+start_time = day
+end_time = day + datetime.timedelta(hours=23, minutes=59, seconds=59)
+
+##############################################################################
+# Send the search query.
+
+results = Fido.search(a.Time(start_time, end_time),
+                      a.Instrument('HMI') & a.vso.Physobs("LOS_magnetic_field"),
+                      a.vso.Sample(60 * u.second))
+
+##############################################################################
+# We will only download the first file for the day. For that we use slicing
+# On the search results.
+
+result = results[0, 0]
+
+##############################################################################
+# Download the file.
+
+downloaded_files = Fido.fetch(result)
+
+##############################################################################
+# As the `fetch` method returns a list, our filename is its first element.
+
+file_name = downloaded_files[0]
+
+##############################################################################
+# Now to download and read the SRS file.
+# Download the SRS file.
+
+srs_results = Fido.search(a.Time(start_time, end_time), a.Instrument('SOON'))
+srs_downloaded_files = Fido.fetch(srs_results)
+
+##############################################################################
+# We get one SRS file per day. So we pass the filename into the SRS reader. So
+# now `srs_table` contains an astropy table.
+
+srs_table = srs.read_srs(srs_downloaded_files[0])
+print(srs_table)
+
+##############################################################################
+# We only need the rows which have 'ID' = 'I' or 'IA'.
+
+if 'I' in srs_table['ID'] or 'IA' in srs_table['ID']:
+    srs_table = srs_table[np.logical_or(
```

Comment:
I would not line break this here in a gallery example. If you really want to line break I would do it after a `,`.